src/Uno.Foundation.Runtime.WebAssembly/Interop/Runtime.wasm.cs

```diff
@@ -200,10 +200,12 @@ private static bool InnerInvokeJSUnmarshalled(string functionIdentifier, IntPtr
 			return res != IntPtr.Zero;
 		}
 
+#pragma warning disable CA2211
 		/// <summary>
 		/// Provides an override for javascript invokes.
 		/// </summary>
 		public static Func<string, string>? InvokeJSOverride;
```

Comment:
@carldebilly Do we need this field at all?

Reply:
I don't know why it's there. It's there since the first public push of the code. Maybe we should check in old private repos to see why it's there. I suspect it was to intercept javascript calls for unit testing.

Reply:
@carldebilly Should we add it to the list of breaking changes?

Reply:
We can make it a property as a breaking change.